commit/operator_withCUDA.cu

```diff
@@ -0,0 +1,639 @@
+#include "operator_withCUDA.cuh"
```

Comment:
Move the file inside `operator`

Reply:
I cannot move these files to the 'operator' folder because COMMIT compiles all the files in this folder using gcc. So, an error raises for the .cu files. I can create another folder called 'operator_cuda'.

Reply:
I agree with the creation of the folder `operator_cuda`